.pre-commit-config.yaml

```diff
@@ -1,4 +1,10 @@
 repos:
+- repo: local
+  hooks:
+    - id: eslint
+      name: eslint
+      language: system
+      entry: sh scripts/eslint.sh
```

Comment:
I found if the script is put under `./frontend` folder, the `pre-commit` cmd can still find it, but won't exec it..